index.js

```diff
@@ -23,13 +23,13 @@ function handleEscaping(tokens, token, index) {
 		return [token];
 	}
 
-	const previousToken = tokens[index - 1];
+	const previousToken = tokens[tokens.length - 1];
 
 	if (!previousToken.endsWith('\\')) {
```

Comment:
Trivial: Removing the negation `!` would make this condition and its effect clearer.